src/shell/lisa_shell

```diff
@@ -245,6 +245,24 @@ echo
 echo
 }
 
+function lisa-check-submods {
+local missing=0
+if [ ! -f ./libs/devlib/setup.py ]; then
+	missing=1
+fi
+if [ ! -f ./libs/bart/setup.py ]; then
+	missing=1
+fi
+if [ ! -f ./libs/trappy/setup.py ]; then
+	missing=1
+fi
```

Comment:
I like that solution... I've suggested Jeol to check for just on library but it's actually more safe to check for all the main dependencies. ;-)